db/009-add-coreID-to-locations.sql

```diff
@@ -0,0 +1,22 @@
+DO $migration$
+    DECLARE migrationId INT;
+    DECLARE lastSuccessfulMigrationId INT;
+BEGIN
+    -- The migration ID of this file
+    migrationId := 9;
+
+    -- Get the migration ID of the last file to be successfully run
+    SELECT MAX(id) INTO lastSuccessfulMigrationId
+    FROM migrations;
+
+    -- Only execute this script if its migration ID is next after the last successful migration ID
+    IF migrationId - lastSuccessfulMigrationId = 1 THEN
+        -- ADD SCRIPT HERE
+        ALTER TABLE locations ADD COLUMN door_coreid text;
```

Comment:
Then how about `door_particlecoreid` and `sensor_particlecoreid`?

I find that there are already a lot of columns in the code base which take a lot of effort to understand. For example, I need to basically write myself out a diagram of all the phone numbers each time to figure out which one is doing what. And I'd like to avoid that in the future, especially for things related to a specific hardware, it makes sense to me to be specific about it. Would we use this same column if we swap out to a different door sensor? I'm thinking probably not. However, if we would, then naming it something more generic like just `door_id` would be more flexible.